extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -269,10 +258,19 @@ public void configurationDescriptorBuilding(
                             getMultiTenancyStrategy(hibernateOrmConfig), false));
         }
 
+        if (!persistenceXmlDescriptors.isEmpty()) {
+            Map<String, List<String>> entityToPersistenceUnit = new HashMap<>();
+            for (String entityClassName : jpaEntities.getAllModelClassNames()) {
+                entityToPersistenceUnit.put(entityClassName, Arrays.asList(PersistenceUnitUtil.DEFAULT_PERSISTENCE_UNIT_NAME)); // use the default entity manager
+            }
+            jpaEntitiesPerPersistenceUnits.produce(new JpaEntityPerPersistenceUnitsBuildItem(entityToPersistenceUnit));
```

Comment:
Yeah, so in this case, we basically can't really say the mapping, so maybe I would just avoid producing the build item.

Especially since it's not obvious you will produce only one of this (you can have XML descriptors and a config persistence unit).